netflix-roulette/src/components/AppHome/AppHome.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { GenresTabs, MoviesList, SearchBar } from "./components";
+import { Box } from "@material-ui/core";
+import { GenreType, MOVIES } from "../../utils";
+
+export interface Movie {
+  id: number;
+  title: string;
+  tagline: string;
+  vote_average: number;
+  vote_count: number;
+  release_date: string;
+  poster_path: string;
+  overview: string;
+  budget: number;
+  revenue: number;
+  genres: string[];
+  runtime: number;
+}
+
+interface AppHomeState {
+  filteredMovies: Movie[];
+  selectedGenre: string;
+}
+
+export class AppHome extends React.Component<{}, AppHomeState> {
+  state = { filteredMovies: MOVIES, selectedGenre: GenreType.ALL };
+
+  handleSearch(searchText: string) {
+    const filteredMovies = MOVIES.filter((movie) =>
+      movie.title.includes(searchText)
```

Comment:
I would say, case-insensitive search would deliver slightly better UX:
```ts
handleSearch(searchText: string) {
    const filteredMovies = MOVIES.filter((movie) =>
      movie.title?.toLowerCase().includes(searchText.toLowerCase())
     );
    this.setState({ filteredMovies });
  }
```

Reply:
Good point. Also added trim to remove extra spaces